packages/core/review-workflows/admin/src/routes/settings/index.tsx

```diff
@@ -162,7 +162,7 @@ export const ReviewWorkflowsListView = () => {
           rows={workflows}
           footer={
             canCreate ? (
-              <TFooter icon={<Plus />} onClick={handleCreateClick}>
+              <TFooter cursor={'pointer'} icon={<Plus />} onClick={handleCreateClick}>
```

Comment:
```suggestion
              <TFooter cursor="pointer" icon={<Plus />} onClick={handleCreateClick}>
```

just a code style thing but we use plain strings if there's no logic (I wish the linter would catch that)

Reply:
Done.